JavaScript/JavaScriptSDK/Context/User.ts

```diff
@@ -11,6 +11,11 @@ module Microsoft.ApplicationInsights.Context {
         public id: string;
 
         /**
+         * Authorized user id
+         */
+        public authId: string;
```

Comment:
I left it public to be inline with all other properties. This is true for user id and session id as well, and they are open.